po/de.po

```diff
@@ -193,6 +203,9 @@ msgid ""
 "also supported by the entry type @book. However, it is advisable to make use "
 "of the dedicated entry type @mvbook."
 msgstr ""
+"Ein mehrbandiges @book. Zur Abwärtskompatibilität sind mehrbandige Bücher "
+"auch unterstützt von Eintragstyp @book. Allerdings ist es empfehlenswert den "
+"dafür bestimmten Eintragstyp @mvbook zu verwenden."
```

Comment:
Hier fällt mir einfach keine gute Übersetzung für `entry type` ein

Reply:
Dokumententyp

Reply:
Change you change it everywhere?